src/components/atoms/InputErrorMessage/CustomText.tsx

```diff
@@ -0,0 +1,50 @@
+import {Text} from 'react-native';
+import {styles} from './styles';
+import Animated, {
+  FadeIn,
+  FadeInUp,
+  FadeOutUp,
+  SlideInUp,
+  SlideOutUp,
+} from 'react-native-reanimated';
+
+export enum Variatns {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary',
+  ERROR = 'error',
+}
+
+type TCustomText = {
+  text: string | undefined;
+  variant: Variatns;
+};
+
+export const CustomText = ({text, variant}: TCustomText) => {
+  const getStyles = () => {
+    switch (variant) {
+      case Variatns.SECONDARY:
+        return styles.secondary;
+      case Variatns.ERROR:
+        return styles.error;
+      case Variatns.PRIMARY:
+      default:
+        return styles.primary;
+    }
+  };
+  const style = getStyles();
```

Comment:
Tworzysz funkcje w komponencie i ją wołasz od razu, bez sensu fest.

Albo po prostu zrób useMemo z tym switchem w środku. Albo możesz zrobić obiekt, gdzie klucz to wariant, a value to styl, i w ogóle wtedy nie bedziesz potrzebował żadnych operacji, tylko referencja do obiektu lol

Reply:
done